_includes/manuals/nightly/3.6_upgrade.md

```diff
@@ -121,6 +121,13 @@ Clean up the yum metadata cache:
 dnf clean metadata
 {% endhighlight %}
 
+Enable the Ruby 2.7 module:
```

Comment:
On line 106 there's a yum erase line. Perhaps that should be updated to remove `rh-ruby25\*` as well?

Reply:
Updated.